kart/tile/importer.py

```diff
@@ -447,9 +448,19 @@ def extract_tile_metadata(self, source):
             local_path = self.repo.lfs_tmp_path / str(uuid.uuid4())
             fetch_from_s3(source, local_path)
             tile_path = local_path
+            # Also fetch sidecar files, if present.
+            for sidecar_file, suffix in self.sidecar_files(source):
+                try:
+                    fetch_from_s3(
+                        sidecar_file,
+                        local_path.with_name(local_path.name + suffix),
+                    )
+                except botocore.exceptions.ClientError as e:
+                    pass  # This kind of sidecar file doesn't exist for this tile. That's okay.
```

Comment:
we should probably check what the error is; don't want to silently ignore bad things. 

https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html 

Also not sure what specific error codes we should look for here